theory.tex

```diff
@@ -38,74 +38,114 @@ \subsubsection{Separative power}
 \end{eqnarray}
 
-Centrifuge parameters, such as average gas temperature, $T$, peripheral speed,
-$v$, height, $h$, diameter, $d$, pressure ratio, $x$, feed flow rate, $F$,
-counter-current flow ratio, $L/P$, and efficiency, $e$ have been chosen (Table
-\ref{tab:centrifuges}) to match the cascade design describe in
-\cite{glaser.2008} and \cite{walker.2017} using P1-type centrifuges.
+In this equation, the parameters of average gas temperature, $T$, peripheral speed,
+$v_a$, height, $h$, diameter, $d$, pressure ratio, $x$, feed flow rate, $F$,
+counter-current flow ratio, $L/F$, are intrinsic to the centrifuge design.
+To match the cascade design describe in \cite{glaser.2008} and \cite{walker.2017},
+P1-type centrifuge properties have been chosen (Table \ref{tab:centrifuges}).
 
 \begin{table}[htb]
     \centering
     \caption{Summary of the centrifuge parameters.}
-    \begin{tabular}{cccccccc}
+    \begin{tabular}{ccccccc}
         \toprule
-        $T$[K] & $v$[m/s] & $h$[m] & $d$[m]   & $x$      & $F$[mg/s]  & $L/F$ & $e$  \\
+        $T$[K] & $v$[m/s] & $h$[m] & $d$[m]   & $x$      & $F$[mg/s]  & $L/F$  \\
         \midrule
-        320    & $320$    & $1.8$  & $0.105$  & $10^{3}$ & $13$       & 2     & 1.0  \\
+        320    & $320$    & $1.8$  & $0.105$  & $10^{3}$ & $13$       & 2   \\
         \bottomrule
     \end{tabular}
     \label{tab:centrifuges}
 \end{table}
 
+The variable $Z_p$ is the rectifier length, or the location of the feed point,
+and has an optimal axial location as defined by \cite{raetz.phd}:
+
+\begin{equation}
+    Z_p = \frac{(1-\theta)(1+L/F)}{1-\theta+L/F}Z
+\end{equation}
+
+This optimizes the recitifer length based on the cut, $\theta$, which is an expression
+of the fraction of the centrifuge feed that is output as product, and the counter-current
+flow, $L/F$. In practice, this value is a design parameter that is defined in the model
+during the design of the centrifuge cascade.
+
+The parameters $r_1$ and $r_2$ are the separation radii of the enriched material
+(here ${}^{235}\mathrm{U}$ vs. ${}^{238}\mathrm{U}$), $r_1$ being the withdrawal
+radius for the lighter isotope and $r_2$ for the heavier isotope. R\"aetz's
+two-shell model looks for optimal values between these two, as described by
+the hydrodynamic equations. The radii ratio is optimized using the following relationship:
+
+\begin{equation}
+    \max \left\{ \left[1-\left(\frac{r_1}{r_2}\right)^2 \right]^2 \times \left[ \ln \left(\frac{r_2}{r_1} \right) \right]^{-1} \right\}
+\end{equation}
+
+This ratio can further be constrained by approximating $r_2$ as being very close
+to the inner centrifuge wall with radius $a$:
+
+\begin{equation}
+    \left(\frac{r_1}{r_2}\right) \approx \left(\frac{r_1}{a}\right) = \sqrt{1 - \frac{2RT}{M}(\ln x)\frac{1}{v_{a}^{2}}}
+\end{equation}
+ 
+Here the gas constants are molar mass, $M$, temperature $T$, universal gas constant, $R$,
+and the pressure ratio, $x$ (typically $1000:1$) \cite{}. %needs citation?
```

Comment:
This comes from `glaser.2008`, page 4. But there is no reference there, not sure if he is referencing Raetz or...?
```suggestion
and the pressure ratio, $x$ (typically $1000:1$) \cite{glaser.2008}. %needs citation?
```